### PR TITLE
added absolute function.

### PR DESCRIPTION
**Pull request by ItzSgtTanker**
Hello all! i decided to add an absolute value function to the calculator as most calculators have this option.
SharedScreenshot

### CODE REVIEW

File: src/simplejavacalculator/UI.java [hunk 1]
```diff
@@ -33,11 +33,11 @@
     private final JTextArea text;
     private final JButton but[], butAdd, butMinus, butMultiply, butDivide,
             butEqual, butCancel, butSquareRoot, butSquare, butOneDevidedBy,
-            butCos, butSin, butTan, butxpowerofy, butlog, butrate;
+            butCos, butSin, butTan, butxpowerofy, butlog, butrate, butabs;
     private final Calculator calc;
 
     private final String[] buttonValue = { "0", "1", "2", "3", "4", "5", "6",
-            "7", "8", "9" };
+            "7", "8", "9", "10" };
```
Comment (pH-7) [inline src/simplejavacalculator/UI.java]: Why did you add "10", here? Anything related to `abs` feature?

Comment (ItzSgtTanker) [reply src/simplejavacalculator/UI.java]: i'm terribly sorry about that. it was late at night, so my tired mind thought each button was assigned a value. however, looking back now, i see they don't. i assume we can discard that portion of the code.

Comment (pH-7) [reply src/simplejavacalculator/UI.java]: No worries @ItzSgtTanker! That happens sometimes 😃

**Comment by pH-7** [comment]
Thanks a lot for your contribution here! I really appreciate it! 😊
